requirements.txt

```diff
@@ -0,0 +1,9 @@
+boto3==1.16.32
```

Comment:
Where did all of these requirements come from?
I usually don't like to freeze versions, maybe we should transition to poetry and specify `^` versions?